packages/story-editor/src/app/highlights/quickActions/mediaUploadWrapper.js

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+/**
+ * Internal dependencies
+ */
+/**
+ * Internal dependencies
+ */
+import { useConfig } from '../..';
+
+const mediaUploadPropsType = {
+  title: PropTypes.string,
+  buttonInsertText: PropTypes.string,
+  onSelect: PropTypes.func.isRequired,
+  onSelectErrorMessage: PropTypes.string,
+  onClose: PropTypes.func,
+  onPermissionError: PropTypes.func,
+  type: PropTypes.oneOfType([
+    PropTypes.string,
+    PropTypes.arrayOf(PropTypes.string),
+  ]),
+  cropParams: PropTypes.object,
+  multiple: PropTypes.bool,
+};
+
+const MediaUploadWrapper = ({ children, mediaUploadProps, ...props }) => {
```

Comment:
This is cool, though do we expect to have multiple uses of `MediaUploadWrapper`? 

I think having it inlined in `useQuickActions.js` would be fine too and reduce one layer of reader indirection. The "meat" of this component is only a couple lines of code.

Reply:
`useQuickActions` is pretty large already (1000+ lines), but we could move it to that file and keep it as a separate component. I'd say this keeps things a little cleaner. Thoughts?

Reply:
But now that I've cleaned it up we may not need the extra file actually